admin-dev/themes/new-theme/scss/components/layout/_nav_bar.scss

```diff
@@ -467,23 +477,23 @@
           }
 
           ul.submenu {
-            position: absolute !important;
-            display: block !important;
-            top: 34px !important;
-            width: 200px !important;
+            position: absolute;
```

Comment:
How do you know if the `!important` needs to be kept or no?
Sometimes you use `/* stylelint-disable declaration-no-important */` and other times you don't Did you have to test each case?

Reply:
I tested case by case, many needs to be kept so i has the disable line, many could be done in another way that we had to refacto it, when I corrected errors I had to test everything I was changing in order to keep the BO working as before, hopefully I did not forget anything